analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyCollectionsShouldNotBeEnumerated.cs

```diff
@@ -208,16 +212,16 @@ public void Methods_Raise_Issue()
         obs[1] = 5;                     // FIXME Non-compliant
 
         var array = new int[0];
-        array.Clone();                  // FIXME Non-compliant
-        array.CopyTo(null, 1);          // FIXME Non-compliant
-        array.GetEnumerator();          // FIXME Non-compliant
-        array.GetLength(1);             // FIXME Non-compliant
-        array.GetLongLength(1);         // FIXME Non-compliant
-        array.GetLowerBound(1);         // FIXME Non-compliant
-        array.GetUpperBound(1);         // FIXME Non-compliant
-        array.GetValue(1);              // FIXME Non-compliant
-        array.Initialize();             // FIXME Non-compliant
-        array.SetValue(5, 1);           // FIXME Non-compliant
+        array.Clone();                  // Noncompliant
+        array.CopyTo(null, 1);          // Noncompliant
+        array.GetEnumerator();          // Noncompliant
+        array.GetLength(1);             // Noncompliant
+        array.GetLongLength(1);         // Noncompliant
+        array.GetLowerBound(1);         // Noncompliant
+        array.GetUpperBound(1);         // Noncompliant
+        array.GetValue(1);              // Noncompliant
+        array.Initialize();             // Noncompliant
+        array.SetValue(5, 1);           // Noncompliant
         _ = array[1];                   // FIXME Non-compliant
         array[1] = 5;                   // FIXME Non-compliant
```

Comment:
These two cases go under S6466 and shouldn't be part of this rule. The rule idea is already captured in the planning document, so we should mark the FIXME comments as `Compliant, should be part of S6466`